src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -528,9 +528,12 @@ void LiveFunctionsDataView::OnDataChanged() {
 }
 
 void LiveFunctionsDataView::OnTimer() {
-  if (app_->IsCapturing()) {
-    OnSort(sorting_column_, {});
+  if (!app_->IsCapturing()) return;
+  for (uint64_t missing_scope_id : FetchMissingScopeIds()) {
+    indices_.push_back(missing_scope_id);
```

Comment:
Can you elaborate on the performance implications? Do we get a problem when we run this every 300ms and the total number of scope ids is large?

Reply:
I've just instrumented 1k functions. The thing is pretty fast. The step I have added takes ~100us. I don't see the reason to bother with trickier structures right now. 

Should I add a comments about that?